docs/reference/query-dsl/text-expansion-query.asciidoc

```diff
@@ -88,13 +88,7 @@ competitively against the current best matching documents. However, the tokens
 generated by the ELSER model don't work well with the Max WAND optimization.
 Consequently, enabling Max WAND can actually increase query latency for
 `text_expansion`. For datasets of a significant size, disabling Max
-WAND leads to lower query latencies.
-
-Max WAND is controlled by the
-<<track-total-hits, track_total_hits>> query parameter. Setting track_total_hits
-to true forces {es} to consider all documents, resulting in lower query
-latencies for the `text_expansion` query. However, other {es} queries run slower
-when Max WAND is disabled.
+WAND leads to lower query latencies. Max WAND is controlled by the <<track-total-hits, track_total_hits>> query parameter.
```

Comment:
This change could be confusing because it's not necessarily clear which `track_total_hits` setting disables Max WAND. The [search API](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-your-data.html#track-total-hits) documentation doesn't mention Max WAND, so the user needs to also read the full blog post to understand how to set `track_total_hits`.

Maybe we also update the search API documentation to explicitly describe which settings enable/disable Max WAND?

Reply:
Good suggestion, thank you! 
<img width="779" alt="image" src="https://github.com/elastic/elasticsearch/assets/63422879/09623b6f-eeef-4de5-9526-be3d776f8b9d">